src/entity/ReputationEvent.ts

```diff
@@ -42,10 +45,21 @@ export const REPUTATION_THRESHOLD = parseInt(
   process.env.REPUTATION_THRESHOLD || '250',
 );
 
-export const canGrantReputation = (
-  grantBy: Pick<User, 'reputation' | 'flags'>,
-): boolean => {
-  return grantBy.reputation >= REPUTATION_THRESHOLD && !grantBy.flags?.vordr;
+const GRANTS_PER_DAY = parseInt(process.env.REPUTATION_GRANTS_PER_DAY || '5');
```

Comment:
Remote config maybe?

Reply:
I don't expect to change it. I was tempted to set it hardcoded to 5 😅